src/utils/generateData.ts

```diff
@@ -386,15 +406,21 @@ export const generateRichItems = async (
 const richItemsGenerate = (
   items: Item[],
   platforms: Platform[],
-  projects: Project[]
+  projects: Project[],
+  departments: Department[],
+  vaults: Vault[]
 ): RichItem[] => {
   const richItems: RichItem[] = []
   for (const item of items) {
     richItems.push({
       ...item,
       id: item.id,
       platform: generateRandomNumber(0, platforms.length),
-      project: generateRandomNumber(0, projects.length)
```

Comment:
note: the vaults and departments tables don't have an integer id - it's a string that's a short version of the actual name.  So, you need to get the row at a random number index, and store the `id` for that row.

Reply:
Oh, hold on.  I see you've tried an alternate strategy for this, but it looks a bit fragile (including if we change what department ids look like, or introduce a third vault.

I'd rather you get a random row, and use the id for that row, please.